### ISSUE TITLE
cron.present is not working correctly.

### ISSUE THREAD

**Issue by steverweber** [saltstack/salt#19916]
latest dev release....
when using mutiple cron jobs states only the last one set is used... this caused our production system to skip a dailly task for a few weeks :(

sls file...:
```
{{ sls }} cron to feed the syslogs to the database:
    cron.present:
        - name: {{ vars.app.path }}/feed_logfile
        - identifier: mailin_feed_logfile
        - user: {{ vars.user.name }}
        - minute: random
        - hour: '17'

{{ sls }} cron to send notice:
    cron.present:
        - name: {{ vars.app.path }}/notify
        - identifier: mailin_notify
        - user: {{ vars.user.name }}
        - minute: random
        - hour: '20'
```

salt minion state.highstate...
```
----------
          ID: mfcf.mailmigrate cron to feed the syslogs to the database
    Function: cron.present
        Name: /srv/mailin/feed_logfile
      Result: True
     Comment: Cron /srv/mailin/feed_logfile added to mailin's crontab
     Started: 14:23:58.369677
    Duration: 59.19 ms
     Changes:   
              ----------
              mailin:
                  /srv/mailin/feed_logfile
----------
          ID: mfcf.mailmigrate cron to send notice
    Function: cron.present
        Name: /srv/mailin/notify
      Result: True
     Comment: Cron /srv/mailin/notify added to mailin's crontab
     Started: 14:23:58.429236
    Duration: 55.714 ms
     Changes:   
              ----------
              mailin:
                  /srv/mailin/notify

Summary
-------------
Succeeded: 19 (changed=2)
```

on minion:
crontab -e
```
# Lines below here are managed by Salt, do not edit
# SALT_CRON_IDENTIFIER:mailin_notify
17 20 * * * /srv/mailin/notify
```
Notice missing feed_logfile job.
It seems that the cron module|state is overwriting the previous salt managed cronjobs.

some grains of minion:
```
    os:
        CentOS
    os_family:
        RedHat
    osarch:
        x86_64
    oscodename:
        Final
    osfinger:
        CentOS-6
    osfullname:
        CentOS
    osmajorrelease:
        6
    osrelease:
        6.6
    pythonversion:
        - 2
        - 6
        - 6
        - final
        - 0
    saltversion:
        2015.2.0-1193-gf365e89
    selinux:
        ----------
        enabled:
            False
        enforced:
            Disabled
```

**Comment by basepi** [comment]
Wow that's no good! We'll definitely investigate this.

**Comment by cachedout** [comment]
Some git bisect suggests that this was introduced here: a8a66fa7ecea4d69855c1b646b18765c88e9ad2f

**Comment by cachedout** [comment]
Fix is in #201386. Thanks, @steverweber. Sorry this bug bit you. :-/

**Comment by steverweber** [comment]
Thanks.  it wasnot so bad bacause Salt helped me find the issue :)
